src/libraries/System.Text.Json/gen/Resources/xlf/Strings.cs.xlf

```diff
@@ -142,6 +142,16 @@
         <target state="translated">Typ obsahuje více členů s komentářem JsonExtensionDataAttribute</target>
         <note />
       </trans-unit>
+      <trans-unit id="TypeContainsRefLikeMemberFormat">
+        <source>The type '{0}' includes the ref like property, field or constructor parameter '{1}'. No source code will be generated for the property, field or constructor.</source>
+        <target state="new">The type '{0}' includes the ref like property, field or constructor parameter '{1}'. No source code will be generated for the property, field or constructor.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="TypeContainsRefLikeMemberTitle">
+        <source>Type includes ref like property, field or constructor parameter.</source>
+        <target state="new">Type includes ref like property, field or constructor parameter.</target>
+        <note />
+      </trans-unit>
```

Comment:
Were the .xlf files changed intentionally?

Reply:
Yes, they are generated automatically every time a new resource string is added.